MinecraftClient/Protocol/Handlers/PacketTypesOut.cs

```diff
@@ -67,5 +67,8 @@ public enum PacketTypesOut
         // Added in 1.19
         ChatCommand,
         ChatPreview,
+
+        // Added in 1.19.1 (1.19.2)
+        MessageAcknowledgment
```

Comment:
Same here (I'd suggest to order the items alphabetically, and, if you want, put a comment on the right stating in which version the packet was introduced.)